docs/deploying/vps.md

```diff
@@ -0,0 +1,146 @@
+
+## Choosing right VPS
+
+Although Jellyfish doesn't have minimum requirements to work, we encourage you to choose a middle-sized VPS as media processing is consuming quite a lot of CPU.
+
+For this tutorial, we decided on CPX21 from Hetzner Cloud running Ubuntu 22:
+
+![Hetzner Cloud - choosing VPS](/img/vps-1.png)
+
+:::tip
+Remember about adding your public SSH while creating a VPS instance. It will ease up connecting to VPS later. If you don't add any key you're going to need a root password (you'll receive it in an email after creating VPS)
+:::
+
+## *cloud-init* config
+
+*cloud-init* is a tool that lets you set up a newly created server with a YAML config file. In the configuration you can create a user, choose packages to be installed, configure them, write or modify files and in our case finally run a docker container.
+
+Full documentation about keys you can use to write _cloud-config_ file can be found in [the official *cloud-init* documentation](https://cloudinit.readthedocs.io/en/latest/reference/modules.html).
+
+We prepared a template *cloud-config* that will prepare your server and start a chosen version of jellyfish:
+
+<details>
+  <summary>cloud-config.yaml</summary>
+
+  ```
+  #cloud-config
+  users:
+    - name: jellyfish
+      groups: docker
+      sudo: null
+      shell: /bin/false
+      no_create_home: true
+  ssh_pwauth: false
+  disable_root_opts: no-port-forwarding,no-agent-forwarding,no-X11-forwarding
+  apt:
+    sources:
+      docker.list:
+        source: "deb [arch=amd64,arm64] https://download.docker.com/linux/ubuntu jammy stable"
+        keyid: 9DC858229FC7DD38854AE2D88D81803C0EBFCD88
+  packages:
+    - ufw
+    - fail2ban
+    - gzip
+    - containerd.io
+    - docker-ce
+    - docker-ce-cli
+  write_files:
+    - path: /opt/jellyfish/env-file
+      defer: true
+      owner: jellyfish:jellyfish
+  runcmd:
+    - export JELLYFISH_VERSION=0.1.0-ed317b
+    - docker pull ghcr.io/jellyfish-dev/jellyfish:$JELLYFISH_VERSION
+    - systemctl enable fail2ban
+    - ufw default deny outgoing
+    - ufw default deny incoming
+    - ufw default deny routed
+    - ufw allow in ssh
+    - ufw allow out https
+    - ufw allow out domain
+    - ufw allow in 80/tcp
+    - ufw allow in 50000:50050/udp
+    - ufw allow out from any port 50000:50050 proto udp to any
+    - ufw enable
+    - |
+      cat << EOF > /opt/jellyfish/env-file
+      SERVER_API_TOKEN=test_token
+      SECRET_KEY_BASE=$(openssl rand -base64 32)
+      INTEGRATED_TURN_IP=$(ip route get 1.0.0.0 | sed -n 's/^.*src \([0-9.]*\) .*$/\1/p')
+      JELLYFISH_ADDRESS=$(ip route get 1.0.0.0 | sed -n 's/^.*src \([0-9.]*\) .*$/\1/p')
+      INTEGRATED_TURN_PORT_RANGE=50000-50050
+      CHECK_ORIGIN=false
```

Comment:
Does disabling `CHECK_ORIGIN` increase risk of some kind of attack?
If I understand correctly, the reverse proxy makes this flag have no effect.
But there is no reverse proxy in this setup